__tests__/auth.test.ts

```diff
@@ -1,6 +1,5 @@
-import * as httpm from '../_out';
```

Comment:
I actually want to test the output that we are going to publish.  Not the root source code.

Reply:
this needs to be out.  see the tsconfig which builds to _out

Reply:
I assumed that `ts-jest` would use the same output as the tsconfig so that this wouldn't be necessary